draft-ietf-httpbis-sfbis.md

```diff
@@ -61,6 +61,13 @@ informative:
   RFC9113:
     display: HTTP/2
   HPACK: RFC7541
+  UNICODE-SECURITY:
+    title: Unicode Security Considerations
+    author:
+    - name: Mark Davis
+    - name: Michel Suignard
+    date: 2014-09-19
+    target: http://www.unicode.org/reports/tr36/
```

Comment:
Should have the MD equivalent of seriesInfo name="Unicode Technical Report" value="#16"